execution_invariance_test/test_folders.txt

```diff
@@ -0,0 +1,2 @@
+execution_invariance_test/projects/simple-test simple_test
+end2end_tests/projects/simple-with-pytest simple_with_pytest
```

Comment:
Why both of them are here?

Reply:
I wanted to check if it worked with multiple entries. Should I remove the end2end test?

Reply:
That's good that you have checked.
I think it would be best to not have the end2end project here.